1.9/administration/component-management.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Component Management
+menu_order: 5.5
+---
+
+The DC/OS Component Package Manager (Pkgpanda) installs and manages DC/OS components on individual DC/OS nodes.
+
+The DC/OS Component Package Manager (Pkgpanda) is not designed to be used by operators of DC/OS but rather by the DC/OS components themselves when performing node lifecycle events, like installing, upgrading, and uninstalling.
+
+DC/OS components are bundled into component packages that are compressed at build time, included as part of the DC/OS installer, transported from the bootstrap machine to the desired node by the installer, and decompressed to a specific location on each node during setup. Generally these component packages contain one or more systemd unit definitions, binaries, and configuration files.
```

Comment:
They're not custom installation specific... The foundation team creates builds at release time.